### ISSUE TITLE
Darkmode reversion errors

### ISSUE THREAD

**Issue by fstutzman** [yujitach/MenuMeters#22]
When a user switches to Dark Mode, textual elements render properly (switching from dark to light colors).

When a user switches from DarkMode, textual elements remain light colors, rendering them invisible.

I've attached a gif of the issue (note - the gif loops, so it makes it look like the text eventually renders - but it does not).

![error](https://cloud.githubusercontent.com/assets/4457653/12756120/44b466a4-c9a0-11e5-9159-1629cb3e5e6d.gif)

**Comment by fstutzman** [comment]
I have fixed this error, if you enable me I can push up a fix branch.

```
mdm:MenuMeters fred$ git diff
diff --git a/Common/MenuMeterWorkarounds.m b/Common/MenuMeterWorkarounds.m
index c6d47f6..8773bbb 100644
--- a/Common/MenuMeterWorkarounds.m
+++ b/Common/MenuMeterWorkarounds.m
@@ -136,15 +136,6 @@ __private_extern__ BOOL IsMenuMeterMenuBarDarkThemed(void) {
     if (interfaceStyle && [interfaceStyle isEqualToString:@"Dark"]) {
                isDark = YES;
        }
-       Class appearanceClass = NSClassFromString(@"NSAppearance");
-       if (appearanceClass && [appearanceClass respondsToSelector:@selector(currentAppearance)]) {
-               id currentAppearance = [appearanceClass performSelector:@selector(currentAppearance)];
-               if (currentAppearance && [currentAppearance respondsToSelector:@selector(name)]) {
-                       if ([[currentAppearance name] hasPrefix:@"NSAppearanceNameVibrantDark"]) {
-                               isDark = YES;
-                       }
-               }
-       }
        return isDark;
 } // IsMenuMeterMenuBarDarkThemed
```

**Comment by yujitach** [comment]
Could you make a pull request on GitHub?

**Comment by fstutzman** [comment]
I did try but did not have permissions to push a branch. Maybe I am missing something.

**Comment by sandhub** [comment]
Hi @fstutzman, when you go to create a `Pull Request`, can you do that by comparing the master branch and the one you created?

**Comment by fstutzman** [comment]
Thanks, got it

**Comment by fstutzman** [comment]
Closed with https://github.com/yujitach/MenuMeters/pull/23

**Comment by fstutzman** [comment]
Reopening this issue as the error is not resolved in 1.9.2.  Will investigate further.

**Comment by fstutzman** [comment]
Argh, it looks like my upgrade to 1.9.2 did not take completely.  Even though the PrefPane was showing 1.9.2, after fully deleting and reinstalling I'm able to confirm the fix works.  Closing this and apologies for any inconvenience - will more thoroughly investigate in the future.
